README.md

```diff
@@ -198,8 +198,8 @@ exactly what and how their test data is configured. These magic values are only
 ## Deployment infrastructure
 
 - review, which has `RAILS_ENV=review`
-- [staging](https://cpd-ecf-staging-web.test.teacherservices.cloud/), which has `RAILS_ENV=staging`
-- [sandbox](https://cpd-ecf-sandbox-web.teacherservices.cloud/), which has `RAILS_ENV=sandbox`
+- [staging](https://st.manage-training-for-early-career-teachers.education.gov.uk/), which has `RAILS_ENV=staging`
+- [sandbox](https://sb.manage-training-for-early-career-teachers.education.gov.uk/), which has `RAILS_ENV=sandbox`
 - [production](https://manage-training-for-early-career-teachers.education.gov.uk/), which has `RAILS_ENV=production`
```

Comment:
Worth linking to the 'internal' URLs too? Could be occasionally handy. No strong feelings either way so feel free to ignore.

```suggestion
- [staging](https://st.manage-training-for-early-career-teachers.education.gov.uk/), which has `RAILS_ENV=staging`, [internal url](https://cpd-ecf-staging-web.test.teacherservices.cloud/)
- [sandbox](https://sb.manage-training-for-early-career-teachers.education.gov.uk/), which has `RAILS_ENV=sandbox`, [internal url](https://cpd-ecf-sandbox-web.teacherservices.cloud/)
- [production](https://manage-training-for-early-career-teachers.education.gov.uk/), which has `RAILS_ENV=production`, [internal url](https://cpd-ecf-production-web.teacherservices.cloud/)
```